app/javascript/components/provider-dashboard-charts/charts_config.js

```diff
@@ -26,9 +29,12 @@ const hourlyPodTimeTooltip = () => (data) => {
   ${theMoment.format('h:mm A')}  ${data[0].value} ${__(data[0].group)} </div>`;
 };
 
-const hourlyTimeTooltip = () => (data) => {
+const hourlyTimeTooltip = (units = undefined) => (data) => {
   const theMoment = moment(data[0].date);
-  const htmlString = `<div class="tooltip-inner">${theMoment.format('h:mm A')}  ${data[0].value} ${data[0].name} </div>`;
+  let htmlString = `<div class="tooltip-inner">${theMoment.format('h:mm A')}  ${data[0].value} </div>`;
+  if (units) {
+    htmlString = `<div class="tooltip-inner">${theMoment.format('h:mm A')}  ${data[0].value} ${__(units)} </div>`;
```

Comment:
And here you can use `LT`. See https://momentjs.com/docs/#/parsing/string-format/ (then scroll to locale aware formats)